source/features/deep-reblame.tsx

```diff
@@ -36,7 +41,7 @@ const getPullRequestBlameCommit = mem(async (commit: string, prNumber: number):
 
 	const associatedPR = repository.object.associatedPullRequests.nodes[0];
 
-	if (!associatedPR) {
+	if (!associatedPR || !repository.file) {
 		return false;
 	}
```

Comment:
I think you'll have to throw instead so in `redirectToBlameCommit` you can output a specific message.
```suggestion
	if (!associatedPR) {
		throw new Error('The PR linked in the title didn’t create this commit');
	}
	
	if (!repository.file) {
		throw new Error('The file was renamed and Refined GitHub can’t find it');
	}
```